### PR TITLE
Updating Beam dependency

### PR DESCRIPTION
**Pull request by sduskis**
Also, depending on guava classes from the Cloud Bigtable repackaged project

### CODE REVIEW

File: bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableConfiguration.java [hunk 1]
```diff
@@ -23,16 +23,16 @@
 import java.util.concurrent.TimeUnit;
 import java.util.Map.Entry;
 
+import org.apache.beam.repackaged.beam_sdks_java_io_hbase.com.google.common.base.Strings;
```
Comment (kevinsi4508) [inline bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableConfiguration.java]: What is so special about Strings?

Comment (sduskis) [reply bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableConfiguration.java]: I missed a commit before I pushed.  I removed `org.apache.beam.repackaged`.  PTAL.

**Comment by sduskis** [comment]
@kevinsi4508, I fixed the imports.  PTAL.

File: bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableIO.java [hunk 2]
```diff
@@ -66,7 +64,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.bigtable.repackaged.com.google.bigtable.v2.SampleRowKeysResponse;
+import com.google.bigtable.repackaged.com.google.common.annotations.VisibleForTesting;
+import com.google.bigtable.repackaged.com.google.common.base.Preconditions;
```
Comment (kevinsi4508) [inline bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableIO.java]: These imports are not in the right order?

**Comment by kevinsi4508** [comment]
LGTM
(check to make sure imports are in the right order)

**Comment by kevinsi4508** [comment]
LGTM

**Comment by byam** [comment]
When will this change going into [bigtable-hbase-beam](https://mvnrepository.com/artifact/com.google.cloud.bigtable/bigtable-hbase-beam) ?
@sduskis

**Comment by sduskis** [comment]
I was hoping to get out a release today, but some things crept up.  At this point, it looks like early next week.

**Pull request merged**